pkg/module/kubebuilder/project.go

```diff
@@ -155,13 +156,23 @@ func (p *Project) DefaultCR(s step.Step) ([]byte, error) {
 	defaultCR := ""
 	if len(d) > 1 {
 		// ask for specific file
-		names := []string{}
-		for _, f := range d {
-			names = append(names, f.Name())
+		var promptString strings.Builder
+		promptString.WriteString(fmt.Sprintf("Please specify the file to use as default CR in %s:\n", samplesDir))
+
+		files := map[int32]string{}
+		for i, f := range d {
+			files[int32(i+1)] = f.Name()
+			promptString.WriteString(fmt.Sprintf("[%d] %s\n", i+1, f.Name()))
+		}
+
+		answer, err := s.Prompt(promptString.String())
+		answerInt, _ := strconv.ParseInt(answer, 10, 32)
```

Comment:
also consider the default input, if press enter, by default choosing the first option so that don't need to type 1, this will improve user experience.